client/cody/package.json

```
@@ -188,7 +202,25 @@
         "command": "cody.guardrails.debug",
         "title": "Cody: Guardrails Debug Attribution",
         "enablement": "config.cody.debug && config.cody.experimental.guardrails && editorHasSelection"
-      }
+      },
+      {
+				"command": "cody.recipe.non-stop",
+				"title": "Cody: Add Fixups with highlighted code",
```

```suggestion
				"title": "Cody: Auto-Apply Fixups",
```

---

On second thought, is this a toggle? In which case, "Toggle" and "mode" is the word that VS Code uses in actions…

```suggestion
				"title": "Cody: Toggle Fixup Auto-Apply Mode",
```

---

Updated to `Cody: Fixup (Experimental)`